src/stores/RelationStore.js

```diff
@@ -34,6 +34,27 @@ const Relation = types
 
       return r && r.children && r.children.length > 0;
     },
+
+    get shouldRender() {
+      const { node1: start, node2: end } = self;
+      const [sIdx, eIdx] = [start.item_index, end.item_index];
+
+      if (start.object !== end.object) return true;
```

Comment:
If one of objects is multiImage, it will also break.
And if objects are different, but currentImage is correct, it should work.
So better to check:
```
if (sIdx !== undefined && start.object.multiImage && 
      sIdx !== start.object.currentImage) return false;
if (eIdx !== undefined && end.object.multiImage && 
      eIdx !== end.object.currentImage) return false;
return true;
```